Makefile

```diff
@@ -177,7 +179,9 @@ test-runtime-image: clean-containers ## Smoke test the runtime image. Pass any d
 build-development:
 	docker build -f $(DEVEL_DOCKERFILE) -t $(DEVEL_IMAGE) --build-arg BUILDER_IMAGE=$(BUILDER_IMAGE) .
 
+development: export LOCAL_USER_ID = $(shell id -u $(USER))
 development: build-development ## Run bash inside the development image
+	docker run -v $(PROJECT_PATH):/home/centos --rm -it -e LOCAL_USER_ID=$(LOCAL_USER_ID) $(DEVEL_IMAGE) make dependencies
```

Comment:
IMO we can call it `USER` and just pass it to docker as `-u`. It works both for user ids and names (so people can pass USER=root).

Reply:
using *-u* could be a solution, but less flexible. GID does not exist in container and UID does not exist in */etc/passwd* which may be handy for some commands.

Reply:
Passing any (for example the same as on local machine) UID to -u will just work and also usernames that exist in the container /etc/passwd (like root). I don't see any issue with it. And calling it `USER` makes it obvious for people using it.

Reply:
For example, in that base image, there exist a user called *default* UID=1001

I can either run container with -u default

```
docker exec -ti -u default apicast_build_0_development_run_1 /bin/bash
bash-4.2$ whoami
default
bash-4.2$ id -u $USER
1001
```

or run using my local machine UID

```
$ docker exec -ti -u `id -u $USER` apicast_build_0_development_run_1 /bin/bash
bash-4.2$ whoami
whoami: cannot find name for user ID 1000
```
I cannot have both things I guess